hyperlink/test/common.py

```diff
@@ -0,0 +1,70 @@
+
+
+from unittest import TestCase
+
+
+class HyperlinkTestCase(TestCase):
```

Comment:
I'd prefer switching to pytest, since this backport runs on _all_ versions of Python.  `_AssertRaisesContext` is not exactly the same as [what's in the standard library](https://github.com/python/cpython/blob/2.7/Lib/unittest/case.py#L109-L128).  I think the differences are irrelevant but it's not a great situation to be in.

But that's for a future PR of course.